ci_framework/roles/libvirt_manager/templates/domain.xml.j2

```diff
@@ -0,0 +1,106 @@
+<domain type='kvm'>
+  <name>cifmw-{{ vm_type }}-{{ vm_id }}</name>
+  <memory unit='GB'>{{ vm_data.value.memory | default(2) }}</memory>
+  <vcpu placement='static'>{{ vm_data.value.cpu | default(2) }}</vcpu>
+  <os>
+    <type arch='x86_64' machine='pc-q35-rhel9.2.0'>hvm</type>
+    <boot dev='hd'/>
+    <bootmenu enable='no'/>
+  </os>
+  <cpu mode='host-passthrough' check='none'/>
+  <clock offset='utc'>
+    <timer name='rtc' tickpolicy='catchup'/>
+    <timer name='pit' tickpolicy='delay'/>
+    <timer name='hpet' present='no'/>
+  </clock>
+  <features>
+    <acpi/>
+    <apic/>
+  </features>
+  <devices>
+    <emulator>/usr/libexec/qemu-kvm</emulator>
+    <disk type='file' device='disk'>
+      <driver name='qemu' type='qcow2'/>
+      <source file='{{ cifmw_reproducer_basedir }}/workload/{{ vm_type }}-{{ vm_id }}.qcow2'/>
+      <target dev='sda' bus='sata'/>
+    </disk>
+    {% for network in vm_data.value.nets %}
+    <interface type='network'>
+      <source network='{{ network }}'/>
+      <mac address='{{ '24:42:%02i'|format(vm_id|int) |random_mac }}'/>
+      <model type="virtio"/>
+    </interface>
+    {% endfor %}
+    <controller type='usb' index='0' model='ich9-ehci1'>
+      <alias name='usb'/>
+      <address type='pci' domain='0x0000' bus='0x00' slot='0x06' function='0x7'/>
```

Comment:
Exact addresses might not be needed, but it is ok.

Reply:
it's an old template I "migrate" from one project to another :]. The less I edit it, the better, to be fair. "it works" is the expected state, and since it works... ;)